src/plugins/workspace/public/components/workspace_creator/workspace_form.tsx

```diff
@@ -0,0 +1,310 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React, { useCallback, useState, FormEventHandler, useRef } from 'react';
+import {
+  EuiPanel,
+  EuiSpacer,
+  EuiTitle,
+  EuiForm,
+  EuiFormRow,
+  EuiFieldText,
+  EuiText,
+  EuiButton,
+  EuiFlexItem,
+  EuiCheckableCard,
+  htmlIdGenerator,
+  EuiFlexGrid,
+  EuiFlexGroup,
+  EuiImage,
+  EuiAccordion,
+  EuiCheckbox,
+  EuiCheckboxGroup,
+  EuiCheckableCardProps,
+  EuiCheckboxGroupProps,
+  EuiCheckboxProps,
+  EuiFieldTextProps,
+} from '@elastic/eui';
+
+interface WorkspaceTemplate {
```

Comment:
@wanglam I see @Hailong-am merged the template changes, should this interface be updated to align with those changes?